tests/overrides/test_data_parallel.py

```diff
@@ -40,16 +38,13 @@ def test_lightning_distributed_module_warn_none_output():
     pl_module.test_step.return_value = None
 
     with pytest.warns(UserWarning, match="Your training_step returned None"):
-        pl_module.training = True
-        pl_module.testing = False
+        pl_module.running_stage = RunningStage.TRAINING
```

Comment:
move the `pl_module.running_stage = ...` out of the `with` and that is nothing expected to happen
also for the three bellow :]

Reply:
I think it is cleaner this way.